modules/ROOT/pages/visual-tour/index.adoc

```diff
@@ -48,9 +48,18 @@ See xref:query/visual-tour.adoc#copilot[Query co-pilot] and xref:explore/explore
 
 image::enable-copilot.png[]
 
+==== Settings for GDS sessions
+
+include::partial$EAP.adoc[]
+
+The following configuration settings are available for link:{gds-sessions-page}[GDS Sessions]:
+
+* The maximum available memory for each Session
+* The maximum number of concurrent Sessions within the organization
+
 == Data Services
 
-This section contains your instances and the Import service.
+This section contains the data services you have access to.
```

Comment:
Is this not the same for all users?

Reply:
It might well be. What would be another way to add Sessions without having to list everything in this sentence? (Another option could be to remove the sentence altogether.)